frontend/src/scenes/billing/BillingGauge.scss

```diff
@@ -23,6 +23,55 @@
             z-index: 100;
         }
     }
+
+    &.BillingGaugeItem--free_tier {
+        // :TRICKY: We want "--border-light" as background color here.
+        // Since this color has an alpha-component, it "stacks" with
+        // other colors. Therefore we're using the effective color
+        // without alpha component as replacement.
+        background-color: #dedede;
+
+        [theme='dark'] & {
+            background-color: #1b1d24;
+        }
+    }
+
+    &.BillingGaugeItem--billing_limit {
+        background-color: #dedede;
+
+        [theme='dark'] & {
+            background-color: #1b1d24;
+        }
+    }
+
+    &.BillingGaugeItem--current_usage {
+        background: var(--danger);
+
+        &.BillingGaugeItem--within-usage-limit {
+            background: var(--brand-blue);
+        }
+    }
+
+    &.BillingGaugeItem--projected_usage {
+        background: repeating-linear-gradient(
+            -45deg,
```

Comment:
Tried to mimic the same stripe pattern we have for funnels / area charts, but changed the angle. Judge yourself:

**Adjusted**
<img width="832" alt="Screenshot 2023-12-08 at 16 21 12" src="https://github.com/PostHog/posthog/assets/1851359/f4af61a6-557a-4724-af31-384943e0411b">


**Funnels/Area Charts**
<img width="833" alt="Screenshot 2023-12-08 at 16 31 26" src="https://github.com/PostHog/posthog/assets/1851359/bda6fc28-cd92-41c4-a155-683fe424cac0">